programs/hydra/src/state.rs

```diff
@@ -29,6 +29,7 @@ pub struct Fanout {
     pub bump_seed: u8,                     //1
     pub account_owner_bump_seed: u8,       //1
     pub total_available_shares: u64,       //8
+    pub payer_reward_basis_points: u64,    //8
```

Comment:
I know you have space because these are all initialized at 300bytes but can we just defined `FANOUT_SIZE: usize = 300` here and use it in `init_parent` so that someone will know if this is a state breaking change easily :)

Reply:
hi again - what timezone are you @jpbogle ?

will have a look when I get settled after the hop. 

I ran into this before when messing with a fork of redacted_j's fairlaunch but hadn't actually considered it as a part of a mainnet-beta thing (with 2k+ instances)

if I add this here does it break those 2k+ instances "_"

Reply:
because the size is 300 there is space left in all those 2k+ instances

but if you place this here, I believe it will cause the following fields that are currently there be offset incorrectly and thus deserialized either incorrectly or fail to deserialize

If you put it at the end of the struct it should work